test/setup/main.tf

```diff
@@ -32,7 +32,7 @@ provider "random" {
 
 module "project_ci_vm" {
   source  = "terraform-google-modules/project-factory/google"
-  version = "~> 7.0"
+  version = "~> 8.0"
 
   name              = "ci-vm-module"
   random_project_id = true
```

Comment:
nit: having `skip_gcloud_download = true` will save us some time while testing

Reply:
nits on nits: I didn't realize PF 9.x was released. I'll bump one version higher as I add the boolean.